packages/projection-typeorm/src/createDataSource.ts

```diff
@@ -204,14 +204,14 @@ const releaseConnection =
       try {
         await connection.queryRunner.rollbackTransaction();
       } catch (error) {
-        logger.error('Failed to rollback transaction', error);
+        logger.warn('Failed to rollback transaction', error);
```

Comment:
Why is this now a warning?

Reply:
This is called every time the observable is unsubscribed or errors. In some cases (e.g. legit stopping the projector), it might not have an active transaction. so this is expected to throw.